src/hooks/usePrismImperativeHandle.tsx

```diff
@@ -46,7 +45,7 @@ export default function usePrismImperativeHandle(
 		download(customFileName?: string) {
 			return new Promise<string>((resolve, reject) => {
 				if (chartView.current) {
-					const filename = `${toSnakeCase(customFileName || title || 'prism_export')}.png`;
+					const filename = `${customFileName || title || 'chart_export'}.png`;
```

Comment:
We don't want to use snake_case and instead should just honor the filename provided.